src/Utils/colors.js

```diff
@@ -1,9 +1,9 @@
 let colors = {
     chatbox: '#4285f4',
-    header: "#546E7A",
-    sidebar:'#263238',
+    header: "#4285f4",
+    sidebar:'#0A518B',
     sidebarElements: "#212121",
```

Comment:
@dynamitechetan, In that case, it's fine. @uday96 if you have any suggestions please provide.